lib/lists.nix

```diff
@@ -6,8 +6,8 @@ rec {
 
   inherit (builtins) head tail length isList elemAt concatLists filter elem genList;
 
-  /*  Create a list consisting of a single element.  `singleton x' is
-      sometimes more convenient with respect to indentation than `[x]'
+  /*  Create a list consisting of a single element.  `singleton x` is
```

Comment:
I don’t see any pro in that notation, but maybe it has an advantage I overlooked. Syntax is the least important aspect of nearly, that’s probably why there’s so much bikeshedding. :)